src/js/components/FilterByService.js

```diff
@@ -93,8 +93,9 @@ var FilterByService = React.createClass({
         dropdownMenuListItemClassName="clickable"
         wrapperClassName="dropdown"
         items={this.getDropdownItems()}
-        onItemSelection={this.handleItemSelection}
         initialID={this.getSelectedId(this.props.byServiceFilter)}
+        onItemSelection={this.handleItemSelection}
+        ref={(ref) => this.dropdown = ref}
```

Comment:
what do you think about this: currently all the components are accessing the dropdown inside of `FilterByService`.

what if we had a method on `FilterByService` called `setDropdownValue` so that the components wouldn't have to access deep properties like `this.healthFilter.dropdown`

instead it would be like `this.healthFilter.setDropdownValue('all')`

this is nice because we can change the `dropdown` name and we'd only need to change it in one place